README.md

```diff
@@ -1,4 +1,7 @@
 <p align="center">
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
This block corrupts the HTML in the README.  I saw this on another PR that I did.  Just removing it should be fine.

Reply:
hmm, are you basing that off the rich-text diff? that view is slightly unreliable, it renders properly when merged: 
[example](https://github.com/logdna/pino-logdna/blob/main/README.md) - [source](https://raw.githubusercontent.com/logdna/pino-logdna/main/README.md)

Reply:
No, not the rich diff within the pr. I loaded the branch from the browser using git and it was messed up there too. That should be a legitimate browser view at that point.

Reply:
fixed, it was weirdly nested but shows up properly now